serving/docker/pytorch-inf2.Dockerfile

```diff
@@ -10,19 +10,20 @@
 # BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied. See the License for
 # the specific language governing permissions and limitations under the License.
 FROM ubuntu:20.04
-ARG djl_version=0.24.0~SNAPSHOT
+ARG djl_version=0.25.0~SNAPSHOT
 ARG torch_version=1.13.1
-ARG python_version=3.8
-ARG neuronsdk_version=2.14.1
-ARG torch_neuronx_version=1.13.1.1.12.0
+ARG python_version=3.9
+ARG neuronsdk_version=2.15.2
+ARG torch_neuronx_version=1.13.1.1.12.1
 ARG transformers_neuronx_version=0.8.268
 ARG neuronx_distributed_version=0.5.0
 ARG neuronx_cc_version=2.11.*
 ARG protobuf_version=3.20.3
-ARG transformers_version=4.34.0
+ARG transformers_version=4.35.0
 ARG accelerate_version=0.23.0
-ARG diffusers_version=0.16.0
+ARG diffusers_version=0.22.0
 ARG pydantic_version=1.10.13
+ARG optimum_source="git+https://github.com/huggingface/optimum-neuron.git@99f080ff42f7c419b81f0e6a4b0a455042d758d0"
```

Comment:
why build from a commits? This could be very unstable

Reply:
Checking to see if I have a stable release to pin to now. If so I will get the versions for optimum-neuron pinned and moved off of this commit.